src/Jobs/OptimizeHtml.php

```diff
@@ -59,4 +71,12 @@ public function hasOptimizers(): bool
         $optimizers = config('page-cache.optimizers', []);
         return is_array($optimizers) && count($optimizers);
     }
+
+    /**
+     * The unique ID of the job.
+     */
+    public function uniqueId(): string
+    {
+        return $this->page->url;
```

Comment:
Just noticed this. Should this be globally unique, or just unique to this specific job?

Reply:
It seems this should be just unique to this specific job